schema.json

```diff
@@ -0,0 +1,1724 @@
+{
```

Comment:
don't need to commit this file, instead the JS script can pull this using an HTTP call.